packages/gatsby-core-utils/src/cpu-core-count.ts

```diff
@@ -48,4 +47,4 @@ const cpuCoreCount = ignoreEnvVar => {
   }
 }
 
-module.exports = cpuCoreCount
+export default cpuCoreCount
```

Comment:
```suggestion
export { cpuCoreCount }
```

And adjust import in https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-core-utils/src/index.js